packages/oidc-middleware/test/e2e/specs/custom-login-page.js

```diff
@@ -20,14 +20,27 @@ browser.waitForAngularEnabled(false);
 describe('Custom login page', () => {
 
   beforeEach(async () => {
+    let cdnUrl='https://ok1static.oktacdn.com/assets/js/sdk/okta-signin-widget/3.0.6';
+
+    if(process.env.NPM_TARBALL_URL) {
+      // Extract the version of sign-in widget from the NPM_TARBALL_URL variable
+      // The variable is of the format https:<artifactory_url>/@okta/okta-signin-widget-3.0.6.tgz
+      let url = process.env.NPM_TARBALL_URL;
+      let dashIndex = url.lastIndexOf('-');
```

Comment:
nit:
I'd play with a little more specific regex or similar to extra the version
- `/okta-signin-widget-[0-9.]+.tgz/`
- `i = url.indexOf('@okta/okta-signin-widget');  version = url.slice(i+1, url.length-4);` (psude-code)